components/Benefits.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @next/next/no-img-element */
+import styles from '/styles/Benefits.module.scss';
+
+type BenefitsProps = {
+    title: string;
+    benefits: string[];
+};
+
+export const Benefits = ({ title, benefits }: BenefitsProps) => {
+    const listOfBenefits = benefits.map((benefit) => (
+        <li className={styles.benefitItem} key={benefit}>
+            <img
+                className={styles.benefitIcon}
+                src="images/benefit-star.svg"
```

Comment:
Not a huge issue, more of a nickpick but the points raised about rendering SVGs from berfore apply here too. Up to you if you want to extract this into it's own "Icon" component or not.